install-dev/index.php

```diff
@@ -44,6 +44,8 @@
 require_once(__DIR__).DIRECTORY_SEPARATOR.'autoload.php';
 
 try {
+    Symfony\Component\Debug\Debug::enable();
```

Comment:
I don't think we should call this method if we're not in debug mode? 🤔

Reply:
Oh thanks you, forget this one :p
FYI, there were a lack of errors reports because of this missing call

Reply:
I still think that we should not call this method in production though

Reply:
I have no idea how to do that, even PHPStan is complaining about using the `_PS_MODE_DEV_` which is always true. Because the installation process is always in dev mode :sweat_smile:

Reply:
I don't think `_PS_MODE_DEV_` is always true 🤔:

https://user-images.githubusercontent.com/2168836/150348094-173af0ef-bd53-4add-aa11-266fd54b5e2f.mov

Reply:
You change the Core code to change the value :sweat_smile: . Without installation, there is no configuration file generated, so the value by default is the `true` ^^

Reply:
@PierreRambaud I don't think so, the file `defines.inc.php` is always present. The generated config file is `app/config/parameters.php`

Reply:
The file is generated after the installation process :sweat_smile:

Reply:
Ok, as you said me on slack, the value of this const is not the same in the production zip, I'll change it!